tolerance-policies/apps/monitor/src/main/java/com/redhat/training/status/StatusService.java

```diff
@@ -0,0 +1,41 @@
+package com.redhat.training.status;
+
+import io.quarkus.logging.Log;
+import javax.enterprise.context.ApplicationScoped;
+import org.eclipse.microprofile.faulttolerance.Retry;
+import org.eclipse.microprofile.faulttolerance.Timeout;
+import org.eclipse.microprofile.faulttolerance.exceptions.TimeoutException;
+
+/**
+ * Simulate a microservice that reads the status of a cloud instance
+ */
+@ApplicationScoped
+public class StatusService {
+
+    private int callCount = 0;
+
+    @Retry( maxRetries = 5, retryOn = TimeoutException.class )
+    public String getStatus() {
+        callCount++;
+
+        delayPossibly();
+
+        return "Running";
+    }
+
+    private void delayPossibly() {
+        long start = System.currentTimeMillis();
+
+        if ( callCount % 5 > 0 ) {
```

Comment:
This took me a while to understand when I first looked at your code. To understand `callCount % 5 > 0`, in my head, I had to go through `1 % 5`, `2 % 5`, ...  `5 % 5`.

To me, flipping the logic and using `callCount % 5 == 0` (or ` ! callCount % 5 == 0` in this particular case) is so much easier to understand (because just looking at the expression, it's immediately obvious to me where this is true).